sheepdog_exporter/sheepdog_exporter.py

```diff
@@ -391,6 +391,10 @@ def is_type_data_category(self, program, project, my_type):
         '''
         data_file_categories = ['data_file', 'metadata_file', 'index_file']
         type_schema = self.get_schema_for_type(program, project, my_type)
+        for property_key in type_schema['properties']:
+            description = type_schema['properties'][property_key].get('description', 'blank')
+            if 'HARMONIZED' in description:
+                print('type:', my_type, 'prop', property_key, description)
```

Comment:
Thanks for the work on this @david4096, I like the approach you took to scripting a solution to the question of which variables are harmonized.

It is nice to see that the list of harmonized variables is colocated in the output, without intervening text. Taking an *excerpt* of the current output as an example:
```
type: medical_history prop type1_diabetes Reported Type-1 Diabetes in the participant (HARMONIZED)

type: medical_history prop type2_diabetes Reported Type-2 Diabetes in the participant (HARMONIZED)

type: medication prop hypertension_meds Take medication for hypertension (HARMONIZED)

type: electrocardiogram_test prop gtc_duration QTC interval (msec) (HARMONIZED)

type: electrocardiogram_test prop pr_duration PR interval (msec) (HARMONIZED)

type: electrocardiogram_test prop qrs_duration QRS interval (msec) (HARMONIZED)

type: electrocardiogram_test prop qt_duration QT interval (msec) (HARMONIZED)

type: blood_pressure_test prop diastolic_blood_pressure Measured diastolic blood pressure (mmhg) (HARMONIZED)

type: blood_pressure_test prop systolic_blood_pressure Measured systolic blood pressure (mmhg) (HARMONIZED)

type: exposure prop smoking_status Smoking condition (HARMONIZED)

type: cardiac_mri prop cardiac_output RV Cardiac output (l/min) (HARMONIZED)
```
It would be more useful to @jessebrennan/I if the property names were dot-formatted and sorted. For example, the above excerpt would instead appear as:
```
Harmonized Variables Found:
blood_pressure_test.diastolic_blood_pressure Measured diastolic blood pressure (mmhg) (HARMONIZED)
blood_pressure_test.systolic_blood_pressure Measured systolic blood pressure (mmhg) (HARMONIZED)
cardiac_mri.cardiac_output RV Cardiac output (l/min) (HARMONIZED)
electrocardiogram_test.gtc_duration QTC interval (msec) (HARMONIZED)
electrocardiogram_test.pr_duration PR interval (msec) (HARMONIZED)
electrocardiogram_test.qrs_duration QRS interval (msec) (HARMONIZED)
electrocardiogram_test.qt_duration QT interval (msec) (HARMONIZED)
exposure.smoking_status Smoking condition (HARMONIZED)
medical_history.type1_diabetes Reported Type-1 Diabetes in the participant (HARMONIZED)
medical_history.type2_diabetes Reported Type-2 Diabetes in the participant (HARMONIZED)
medication.hypertension_meds Take medication for hypertension (HARMONIZED)
```
Sorting helps in multiple ways:
* Provides consistent output between runs (for the same project and across projects), given that the order the types are processed varies between runs (even for the same project)
* Facilitates (semi-manually) creating a consolidated list harmonized variables across the GTEx and all TOPMed projects, which is important due differences in the variables reported by different projects (for example TOPMed COPD-boston-EO includes `medical_history.asthma` but TOPMed SAGE does not)
* Easier comparison with the multiple code segments that contain similar property lists, which are (typically) sorted.

This would probably be sufficient for current purposes.
A wishlist would include a script that automatically produced a consolidated list of harmonized variables from across all Gen3 projects (GTEx and all TOPMed).
Dreaming on includes augmenting this consolidated list with the list of projects that each variable occurred in.